tests/test.py

```diff
@@ -74,14 +75,32 @@
 logger.addHandler(handler)
 logger.setLevel(logging.DEBUG)
 
-DB_HOST = os.environ.get('TEST_DB_HOST', '127.0.0.1')
+DB_HOST = os.environ.get('TEST_DB_HOST', 'localhost')
```

Comment:
what's the difference?

Reply:
use domain name localhost that used as common name in tls keys